src/geom/SkyWcs.cc

```diff
@@ -94,181 +170,328 @@ Angle SkyWcs::getPixelScale(Point2D const& pixel) const {
     // This cross product computes the distance *through* the unit sphere, rather than over its surface,
     // but the difference should be negligible for pixels of sane size
 
-    double pixelAreaSq = square(skyDx[1] * skyDy[2] - skyDx[2] * skyDy[1]) +
-                         square(skyDx[2] * skyDy[0] - skyDx[0] * skyDy[2]) +
-                         square(skyDx[0] * skyDy[1] - skyDx[1] * skyDy[0]);
-    return std::pow(pixelAreaSq, 0.25) * radians;
+    double skyAreaSq = square(skyDx[0] * skyDy[1] - skyDx[1] * skyDy[0]) +
+                       square(skyDx[1] * skyDy[2] - skyDx[2] * skyDy[1]) +
+                       square(skyDx[2] * skyDy[0] - skyDx[0] * skyDy[2]);
```

Comment:
Good suggestion. I had been thinking the same thing, but worried it would be too messy to create Eigen vector versions of `skyDx` and `skyDy`, which are instances of `Point3D`. I took another look at Point3D has an `asEigen` method, so it is trivial..